lte/gateway/python/magma/mobilityd/serialize_utils.py

```diff
@@ -47,8 +47,26 @@ def _desc_state_proto_to_str(proto):
         IPDesc.REAPED: ip_descriptor.IPState.REAPED,
```

Comment:
Not related to your changes since this already exist, but since this functions will be called to convert each IP state in the mobility store objects, can we define the proto / type maps outside the function since anyway these will not change? Same for the IP desc conversion functions.

Reply:
I would keep this change separate, since it is not related to this patch.